### PR TITLE
Add demo for using KFP lightweight components using func_to_container_op

### PR DESCRIPTION
**Pull request by ca-scribner**

### CODE REVIEW

**Comment by blairdrummond** [comment]
Can we split this up into a couple of notebooks? I like the content, but I think it's a bit overwhelming as one document. I think it could be broken up into

- `1-Simple Example`  ( keep it very short, not too wordy. Picture is worth 1000 words ). They should be able to run it and have a *rough* idea of what just happened.

- `2-Advanced Example` (more explanation here, but still skip the gotchas)

- `3-Gotchas`

I think it's good to make each notebook a bit more "bite-sized". (Obviously, I am sometimes guilty of this too)

**Comment by ca-scribner** [comment]
Yeah I like that suggestion.  

Since we don't have any of these right now, how about we push this now as is and I'll break them into bite sized pieces later this week?   I have more small basic tutorials to add later this week anyway so I can do the reorg at the same time.

**Comment by blairdrummond** [comment]
Ok, sure.

**Comment by blairdrummond** [comment]
Epic: https://github.com/StatCan/daaas/issues/57

**Comment by blairdrummond** [comment]
Before the merge, can you roll up the commits into one and give a [semantic commit message](https://seesparkbox.com/foundry/semantic_commit_messages)? You can either squash the commits I guess or do a `git reset --soft` and go from there

**Comment by ca-scribner** [comment]
@blairdrummond squashed

**Comment by blairdrummond** [comment]
Looks fab.

**Comment by ca-scribner** [comment]
closes StatCan/daaas#39

**Comment by ca-scribner** [comment]
resolve StatCan/daaas#39
